src/neuroconv/datainterfaces/behavior/deeplabcut/_dlc_utils.py

```diff
@@ -341,21 +326,21 @@ def _write_pes_to_nwbfile(
 
 def add_subject_to_nwbfile(
```

Comment:
We should make these functions private I think. We are using them to convert and the thing that should be exposed is our interface. 

I am saying this because technically we are broken backwards comp by changing the name of the file, the right way would be to introduce a soft deprecation, create a new argument, etc.  But I don't think we should be that careful with these ones ... to avoid this problem in the future let's make this private now.

What do you think?

Reply:
To be honest, I would get rid of `add_subject_to_nwbfile` entirely and unpack its contents into the interface (which would still not break backwards compatibility of the interface). It adds another redirection that seems completely unnecessary.

What do you think?

Reply:
Nm, too much work for just an aesthetic improvement (although I do dislike excessive nesting/redirection).